charts/prometheus-postgres-exporter/templates/podsecuritypolicy.yaml

```diff
@@ -0,0 +1,39 @@
+{{- if .Values.rbac.pspEnabled }}
```

Comment:
optional choice here, but as part of the bigger psp changes I added a global var to our config called `pspEnabled` true|false (no rbac in the name) . You could tie in to that one rather than the local one here.

Reply:
i'll follow up on this point before merge, thank you

Reply:
Hey actually I don't think it's on master yet, so I'll omit that for now